documentation/onboard-service.md

```diff
@@ -79,7 +83,14 @@ Amend the AD group of the area:
 - The developers should now have access to continue with the set-up
 
 ## Deploy new service
-In the service repository, runs the Makefile commands.
+
+### Configure environment variables
+The application is configured for each environment using [environment variables](https://12factor.net/config). The [application_configuration](https://github.com/DFE-Digital/terraform-modules/tree/main/aks/application_configuration) module allows setting the variables and mapping them to the application automatically. Internally, *non-secret variables* are stored in a Kubernetes configmap and *secret variables* are stored in a Kubernetes secret.
+
+- Default rails variables are provided by [the terraform module](https://github.com/DFE-Digital/terraform-modules/blob/2fe7c6330db16abc2097cda37ff369c22b398f61/aks/application_configuration/resources.tf#L3)
+- Access keys to resources created by terraform like databases or storage accounts are [configured directly via terraform](https://github.com/DFE-Digital/teacher-services-cloud/blob/a14bc31bad011d3227c9787b1d60011431359373/templates/new_service/terraform/application/application.tf#L19)
+- Custom secrets are manually configured in the application key vault created in [Create terraform environment](#create-terraform-environment). e.g. rails secret key, notify key...
```

Comment:
do we mention the app key vault is loaded via https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/terraform/application/application.tf#L9

Reply:
rephrased, thanks